boilerplate/fullstack/remix/app/routes/supertokens.sessioninfo.$.tsx

```diff
@@ -0,0 +1,15 @@
+import { json } from "@remix-run/node";
+import { LoaderFunctionArgs } from "react-router-dom";
+import { getSessionDetails } from "../lib/superTokensHelpers.js";
+
+// Loader function for handling GET requests that also adds cache control headers
+export async function loader({ request }: LoaderFunctionArgs) {
+    try {
+        const res = await getSessionDetails(request);
```

Comment:
- we need to check if the res.RemixResponse is !== undefined, and if it's not, we need to return that instead.
- See how we have created a function like `withSession` in the nextjs app. Create a similar function for remix. This function is responsible for a bunch of things:
   - Sending a response for 401 if the access token is missing / expired
   - If the user's API logic updates the access token payload, we need to update the response headers. 
   - Catch session related errors using our error handler and return the response via a remix response

Reply:
✔️

Reply:
✔️